egs2/TEMPLATE/sv1/db.sh

```diff
@@ -0,0 +1,7 @@
+# Set the path of your corpus
```

Comment:
Because the task is different? Feel free to change it. We can also add some datasets in current `egs2/TEMPLATE/sv1/db.sh` to `egs2/TEMPLATE/asr1/db.sh`, but I thought it was better to split since the task is not ASR and we don't have transcriptions for VoxCelebs or other speaker recognition datasets.